src/MqttController.js

```diff
@@ -7,30 +6,28 @@ var client  = mqtt.connect('ws://127.0.0.1:9001')
 //assigned to localhost for now.
 client.subscribe('/smartcar/camera') 
 
-function imageStream(message) {
-
-  let ctx = document.getElementById("img").getContext("2d")
-  let imageData = ctx.createImageData(320, 240)
-  let messageLenght = 320 * 240
-  let ci = 0
-  for (ci = 0; ci < messageLenght; ++ci) {
-    let a = 255;
-    let r = message[3 * ci];
-    let g = message[3 * ci + 1];
-    let b = message[3 * ci + 2];
-    imageData.data[4 * ci] = r
-    imageData.data[4 * ci + 1] = g
```

Comment:
Such a change should probably be done on another branch related to mqtt/arduino fixes, same as above